index.html

```diff
@@ -6,6 +6,9 @@
 <title>Spirit Journey Formation Anniversary</title>
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <link rel="stylesheet" href="css/main.css">
+<script defer src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js"></script>
+<script defer src="https://cdnjs.cloudflare.com/ajax/libs/humanize-duration/3.10.0/humanize-duration.min.js"></script>
+<script defer src="js/listdates-min.js"></script>
```

Comment:
TL;DR: defer makes scripts not be render-blocking, but they (should) excecute in source order. async makes scripts not be render-blocking, but they'll execute as soon as they load, regardless of source order.